coldfront/templates/common/navbar_admin.html

```diff
@@ -3,6 +3,7 @@
   <div class="dropdown-menu">
     <a class="dropdown-item" href="/admin">MyBRC Administration</a>
     <a id="navbar-user-search" class="dropdown-item" href="{% url 'user-search-home' %}">User Search</a>
+    <a class="dropdown-item" href="{% url 'user-search-all' %}">All Users</a>
```

Comment:
FYI (no action needed): If we wanted this to be available to non-admin staff (which we don't), we'd also want to add this to `navbar_nonadmin_staff.html`.